docs/API_DESIGN.md

```diff
@@ -606,33 +607,16 @@ CLASS DocumentMetadata
 
 ```pseudocode!
 CLASS DidJwk
-  PUBLIC DATA did: Did
-  PUBLIC DATA document: Document
-  CONSTRUCTOR(public_jwk: Jwk)
-  CONSTRUCTOR(uri: string)
+  STATIC METHOD create(options: DidJwkCreateOptions): BearerDid
   STATIC METHOD resolve(uri: string): ResolutionResult
```

Comment:
Is the intention to also change did:dht and did:dht to just have static `create` and remove constructors?

Reply:
yes, as well as did:web